packages/cursorless-org-docs/src/docs/user/release-notes/0.28.0.md

```diff
@@ -1,7 +1,6 @@
 ---
 version: "0.28.0"
 release_date: "2023-09-01"
-sidebar_position: -2
```

Comment:
Having to change all of these every time we add a release notes is somewhat of a hassle